payplug/__init__.py

```diff
@@ -422,40 +422,18 @@ class InstallmentPlan:
     """
     A DAO for resources.InstallmentPlans which provides a way to query installment plans.
     """
-    @staticmethod
-    def create(**data):
-        """
-        Create an installment plan.
-
-        :param data: data required to create an installment plan
-
-        :return: The installment plan
-        :rtype resources.InstallmentPlan
-        """
-        http_client = HttpClient()
-        response, _ = http_client.post(routes.url(routes.INSTALLMENT_PLANS), data)
-        return resources.InstallmentPlan(**response)
-    
-    @staticmethod
-    def update(installment_plan, **data):
-
-        if isinstance(installment_plan, resources.InstallmentPlan):
-            installment_plan = installment_plan.id
-
-        http_client = HttpClient()
-        response, _ = http_client.patch(routes.url(routes.INSTALLMENT_PLANS, resource_id=installment_plan), data)
-        return resources.InstallmentPlan(**response)
```

Comment:
Why remove these two methods? Any reason for only keeping the `GET` route?

Reply:
A misunderstanding on my part from a previous comment, the next commit should fix it